scripts/devnet/lotus.dockerfile

```diff
@@ -19,7 +19,7 @@ RUN CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__" \
     FFI_USE_GPU="0" \
     make 2k && strip lotus*
 
-FROM ubuntu:22.04
+FROM ubuntu:24.04
```

Comment:
I think `ubuntu:24.04` needs to be paired with `FROM golang:1.21-bookworm AS lotus-builder`